Autopilot/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -20,7 +20,7 @@ void fetchInstructionsMode::execute(attitudeManager* attitudeMgr)
 
     // The support is also here for sending stuff to Path manager, but there's nothing I need to send atm.
 
-    attitudeMgr->setState(sensorFusionMode::getInstance());
+    // attitudeMgr->setState(sensorFusionMode::getInstance());
```

Comment:
same sort of thing. We can't merge changes that comment out previous changes like this, It breaks the flow. Gotta create a dummy implementation for them rather than do this.